littlelight.py

```diff
@@ -188,7 +199,32 @@ def charCreate(save, player):
                         from models import storage
                         player = Player()
                         save = storage.load_save()
-                        player.updateStats(json.dumps(save[0]))
+                        print('save', save)
+                        player.updateStats(json.dumps(save.get('save_1')))
+                        print(player.getStats())
+                        done = startGame(gameDisplay, player, clock, save)
+                        if done:
+                            pygame.quit()
+                            quit()
+                if save_02.rect.collidepoint(pos):

```

Comment:
same here, it might be simpler to have this code condensed, with a variable name determining which save is loaded